meta-rdk-mtk-bpir4/recipes-connectivity/openthread/otbr-agent_4.4.4.0.bb

```diff
@@ -0,0 +1,128 @@
+DESCRIPTION = "OpenThread Border Router Agent"
+HOMEPAGE = "https://github.com/openthread/ot-br-posix"
+
+
+LICENSE = "BSD-3-Clause"
```

Comment:
Probably should be "Proprietary & BSD-3-Clause" because SRC_URI points to gecko_sdk as well as ot-br-posix.